--- a/vendor/league/config/src/Configuration.php	2022-06-01 16:24:02.054851400 +0200
+++ b/vendor/league/config/src/Configuration.php	2022-06-01 16:24:32.786456400 +0200
@@ -65,11 +65,13 @@
      *
      * @psalm-allow-private-mutation
      */
-    public function addSchema(string $key, Schema $schema): void
+    public function addSchema(string $key, Schema $schema, bool $overwrite = true): void
     {
         $this->invalidate();
 
-        $this->configSchemas[$key] = $schema;
+        if ($overwrite || !isset($this->configSchemas[$key])) {
+            $this->configSchemas[$key] = $schema;
+        }
     }
 
     /**
